adapters/bidmachine/bidmachinetest/exemplary/simple-banner.json

```diff
@@ -0,0 +1,78 @@
+{
+    "mockBidRequest":
+    {
+        "id": "test-request-id",
+        "imp": [
+        {
+            "id": "test-impression-id-1",
+            "banner":
+            {
+                "format": [
+                {
+                    "w": 320,
+                    "h": 50
+                }]
+            },
+            "ext":
+            {
+                "bidder":
+                {
+                    "seller_id": "1",
+                    "host": "api-eu",
+                    "path": "auction/rtb/v2"
+                }
+            }
+        }]
+    },
+    "httpCalls": [
+    {
+        "expectedRequest":
+        {
+            "uri": "http://api-eu.example.com/prebid/auction/rtb/v2/1",
+            "body":
+            {
+                "id": "test-request-id",
+                "imp": [
+                {
+                    "id": "test-impression-id-1",
+                    "banner":
+                    {
+                        "format": [
+                        {
+                            "w": 320,
+                            "h": 50
+                        }]
+                    },
+                    "ext":
+                    {
+                        "bidder":
+                        {
+                            "seller_id": "1",
+                            "host": "api-eu",
+                            "path": "auction/rtb/v2"
+                        }
+                    }
+                }]
+            }
+        },
+        "mockResponse":
+        {
+            "status": 200,
+            "body":
+            {
+                "id": "test-response-id",
+                "seatbid": [
+                {
+                    "bid": [
+                    {
+                        "id": "test-bid-id-1",
+                        "impid": "test-impression-id-1",
+                        "price": 1
+                    }],
+                    "seat": "bidmachine"
+                }]
+            }
+        }
+    }],
+    "expectedBidResponses": []
```

Comment:
Why do the tests always have an empty `expectedBidResponses`?

Reply:
Fixed it on all tests